Code/Editor/EditorViewportSettings.cpp

```diff
@@ -588,4 +605,15 @@ namespace SandboxEditor
     {
         AzToolsFramework::SetRegistry(CameraFarPlaneDistanceSetting, aznumeric_cast<double>(distance));
     }
+
+    float CameraDefaultFovRadians()
+    {
+        return aznumeric_caster(AzToolsFramework::GetRegistry(CameraFovSetting, 90.0));
```

Comment:
update: This definitely seems wrong sorry, if we're returning radians, the default fov shouldn't be `90.0` (that's in degrees)